### PR TITLE
SUR-223 Implement Search Component in Force UI as per Figma - Fixed accessibility issues

### PR DESCRIPTION
**Pull request by JelenaTakac**
### Screenshots

Video: https://bsf.d.pr/v/dZ7Uzl

### Checklist:

-   [ ] My code is tested
-   [ ] My code passes the PHPCS tests
-   [ ] I've created the npm build.
-   [ ] My code follows accessibility standards 
-   [ ] My code has proper inline documentation 
-   [ ] I've included any necessary tests 
-   [ ] I've included developer documentation 
-   [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/search/search.jsx [hunk 1]
```diff
@@ -144,7 +168,7 @@ const SearchBoxInput = forwardRef(
 
 		return (
 			<div
-				tabIndex={ 0 }
+				// tabIndex={ 0 }
```
Comment (vrundakansara) [inline src/components/search/search.jsx:171]:
```suggestion
```

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: src/components/search/search.jsx [hunk 2]
```diff
@@ -75,6 +76,29 @@ const SearchBox = forwardRef(
 			dismiss,
 		] );
 
+		useEffect( () => {
+			const handleKeyDown = ( event ) => {
+				if ( event.key === '/' ) {
```
Comment (jaieds) [inline src/components/search/search.jsx:81]: Checking only the `/` key might cause some issues. Instead, we can combine `meta key`+ `/`.

**Review by BabuTechBsf — APPROVED** [review]
Works

**Review by ravindra114 — APPROVED** [review]

File: src/components/search/search.jsx (hunk 2 above)
Comment (JelenaTakac) [reply src/components/search/search.jsx]: Resolved.

File: src/components/search/search.jsx [hunk 3]
```diff
@@ -75,6 +76,30 @@ const SearchBox = forwardRef(
 			dismiss,
 		] );
 
+		useEffect( () => {
+			const handleKeyDown = ( event ) => {
+				// Check if the Meta (command/windows) key and '/' are pressed together
+				if ( event.key === '/' && ( event.metaKey || event.ctrlKey ) ) {
```
[the remark this answers is not in the document]
Comment (JelenaTakac) [reply src/components/search/search.jsx]: Implemented OS detection.

**Review by jaieds — APPROVED** [review]
Working fine.

https://d.pr/v/DBtlPi

**Pull request merged**